webpack.config.js

```diff
@@ -103,6 +109,11 @@ module.exports = {
         railsTransformer("erb"),
         "sass-loader",
       ]
+    },
+
+    {
+      test: path.resolve(__dirname, 'node_modules/normalize.css'),
+      use: 'null-loader',
```

Comment:
WFE doesn't work with normalize css :( it break the UI completely, I guess that has been forgotten in the beginning, we need to allocate some effort adding it.